backend/src/graphql/resolver/TransactionResolver.ts

```diff
@@ -248,26 +246,22 @@ async function calculateAndAddDecayTransactions(
 
     if (previousTransaction) {
       const currentTransaction = userTransaction
-      const decay = await calculateDecayWithInterval(
+      const decay = await calculateDecay(
         previousTransaction.balance,
         previousTransaction.balanceDate,
         currentTransaction.balanceDate,
       )
       const balance = previousTransaction.balance - decay.balance
 
-      if (
-        decayStartTransaction &&
-        decayStartTransaction.received < currentTransaction.balanceDate
-      ) {
+      if (CONFIG.DECAY_START_TIME < currentTransaction.balanceDate) {
         finalTransaction.decay = decay
         finalTransaction.decay.balance = roundFloorFrom4(balance)
         if (
-          decayStartTransaction &&
-          previousTransaction.transactionId < decayStartTransaction.id &&
-          currentTransaction.transactionId > decayStartTransaction.id
+          previousTransaction.balanceDate < CONFIG.DECAY_START_TIME &&
+          currentTransaction.balanceDate > CONFIG.DECAY_START_TIME
```

Comment:
Yeah! Get rid of the scary id arithmetic

Reply:
Yes, but I did not test this yet - i am not 100% this does not have sideeffect I did not understand. For me it made no sense to compare the ids on the one hand and the date on the other. We will see how this will hold up after a test